test/built-ins/RegExp/S15.10.2.12_A2_T1.js

```diff
@@ -0,0 +1,534 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: |
+    The production CharacterClassEscape :: S evaluates by returning
+    the set of all characters not included in the set returned by
+    CharacterClassEscape :: s
+es5id: 15.10.2.12_A2_T1
+description: WhiteSpace
+---*/
+
+var i0 = "";
+for (var j = 0x0000; j < 0x0400; j++)
+  i0 += String.fromCharCode(j);
+var o0 = "\u0009\u000A\u000B\u000C\u000D\u0020\u00A0";
+if (i0.replace(/\S+/g, "") !== o0) {
+  $ERROR("#0: Error matching character class \S between character 0 and 3ff");
+}
+
+var i1 = "";
+for (var j = 0x0400; j < 0x0800; j++)
+  i1 += String.fromCharCode(j);
+var o1 = "";
+if (i1.replace(/\S+/g, "") !== o1) {
+  $ERROR("#1: Error matching character class \S between character 400 and 7ff");
+}
+
+var i2 = "";
+for (var j = 0x0800; j < 0x0C00; j++)
+  i2 += String.fromCharCode(j);
+var o2 = "";
+if (i2.replace(/\S+/g, "") !== o2) {
+  $ERROR("#2: Error matching character class \S between character 800 and bff");
+}
+
+var i3 = "";
+for (var j = 0x0C00; j < 0x1000; j++)
+  i3 += String.fromCharCode(j);
+var o3 = "";
+if (i3.replace(/\S+/g, "") !== o3) {
+  $ERROR("#3: Error matching character class \S between character c00 and fff");
+}
+
+var i4 = "";
+for (var j = 0x1000; j < 0x1400; j++)
+  i4 += String.fromCharCode(j);
+var o4 = "";
+if (i4.replace(/\S+/g, "") !== o4) {
+  $ERROR("#4: Error matching character class \S between character 1000 and 13ff");
+}
+
+var i5 = "";
+for (var j = 0x1400; j < 0x1800; j++)
+  i5 += String.fromCharCode(j);
+var o5 = "\u1680";
+if (i5.replace(/\S+/g, "") !== o5) {
+  $ERROR("#5: Error matching character class \S between character 1400 and 17ff");
+}
+
+var i6 = "";
+for (var j = 0x1800; j < 0x1C00; j++) {
+  if (j === 0x180E) {
+    continue;
+  }
+  i6 += String.fromCharCode(j);
+}
```

Comment:
This if above is the only new part on this file where I skip U+180E as it has changed.